File: original src/Polarimeter REV300 read/Polarimeter/frmNewMeasurement.vb
Public Class frmNewMeasurement

    Dim mSampleName As String
    Dim mAverageNumber As Integer
    Dim mNumberOfRepeatation As Integer

    ReadOnly Property SampleName() As String
        Get
            Return mSampleName
        End Get
    End Property

    ReadOnly Property RepeatNumber() As Integer
        Get
            Return mAverageNumber
        End Get
    End Property

    ReadOnly Property NumberOfRepeatation() As Integer
        Get
            Return mNumberOfRepeatation
        End Get
    End Property

    Private Sub btnCancel_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles btnCancel.Click
        Close()
    End Sub

    Private Sub btnOK_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles btnOK.Click
        mSampleName = txtSampleName.Text
        mAverageNumber = numAverageNumber.Value
        mNumberOfRepeatation = numRepeatation.Value
        Close()
    End Sub

    Public Function Verify() As Boolean
        If txtSampleName.Text = "" Then
            Return False
        Else
            Return True
        End If
    End Function

End Class

File: Polarimeter2019/frmNewMeasurement.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;

namespace Polarimeter2019
{
    public partial class frmNewMeasurement : Form
    {
        private string mSampleName;
        int mAverageNumber;
        int mNumberOfRepeatation;
    
        public string SampleName { get { return txtSampleNamee.Text; } }
        public decimal RepeatNumber { get { return numAverageNumber.Value; } }
        public decimal OfRepeatation { get { return numRepeatation.Value; } }

        public frmNewMeasurement()
        {
            InitializeComponent();
        }

        private void btnOK_Click(object sender, EventArgs e)
        {
            this.DialogResult = DialogResult.OK;
            this.Close();
        }

        private void btnCancel_Click(object sender, EventArgs e)
        {
            this.Close();
        }

        public bool Verify()
        {
            if (txtSampleNamee.Text == "")
                return false;
            else
                return true;
        }
    }
}
